Sources/CorePayments/Networking/APIClient.swift

```diff
@@ -51,12 +52,26 @@ public class APIClient {
         return try await http.performRequest(httpRequest)
     }
     
-    // TODO: - Add GraphQL equivalent request type & function
-    // public func fetch(request: GraphQLRequest) async throws -> HTTPResponse { }
+    /// :nodoc:
+    public func fetch(request: GraphQLRequest) async throws -> HTTPResponse {
+        let url = try constructGraphQLURL(queryName: request.queryNameForURL)
+
```

Comment:
not sure if we need this extra space but up to you:
```suggestion
```

Reply:
I did it intentionally to group `postBody` and `postData` together, but down to remove this newline if it's harder to read